lib/ViewMetaData/ViewMetaData.js

```diff
@@ -26,7 +26,9 @@ const ViewMetaData = ({
       return;
     }
 
-    mutator.updaters.GET();
+    mutator.updaters.GET().catch(() => {
+      // Silently handle fetch errors that may occur during tenant switching
+    });
```

Comment:
In the context of the tenant switch, suppressing the error completely makes perfect sense, but this fetch certainly runs at other times too. I wonder if a `console.warn()` would be better than complete suppression in those circumstances.